src/components/Header/Header.test.tsx

```diff
@@ -128,3 +145,124 @@ describe("Header", () => {
     expect(header).toMatchSnapshot();
   });
 });
+
+describe("Patron is logged in", () => {
+  beforeAll(async () => {
+    jest.spyOn(Cookies, "get").mockReturnValue(mockCookieValue);
+
+    // This is assuming that the sitewide alerts fetch call runs
+    // before the patron information call. If the alerts call runs
+    // after the patron information call, the wrong response will be
+    // returned for each and this test will fail.
```

Comment:
Can you add a `TODO` after this comment that says something like "fix the indeterministic fetch calls".